docs/cimpl6ToolingReference.md

```diff
@@ -400,13 +437,13 @@ The content of the /out directory depends on which exporters were selected to ru
 * [json-schema](#json-schema-export) - this directory contains schemas for the (**TO DO -- need to define the JSON schema export**)
 * [modeldoc](#model-documentation-export) - this directory contains files that present the model with a look and feel similar to [Javadoc](#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html)
```

Comment:
Broken link. Remove the '#' from '#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html'

Reply:
fixed.